src/docker-osx-dev

```diff
@@ -772,6 +774,18 @@ function watch {
   done
 }
 
+#
+# Usage: prepare_sync
+#
+# Configure your development paths prior to sync (or watching) to ensure the
+# appropriate files are excluded and/or included.
+#
+function prepare_sync {
+  configure_paths_to_sync "$docker_compose_file" "${paths_to_sync[@]}"
```

Comment:
These are all local variables in the code, so would this function actually work?